css/includes/components/_networkport.scss

```diff
@@ -56,15 +56,39 @@
     }
 
     &.trunk {
-        background-color: mediumaquamarine;
+        @if var($is-dark) {
```

Comment:
I'm not sure the $is-dark variable works in the GLPI CSS anymore on `main` since we split the theme CSS, GLPI CSS, and Tabler/Bootstrap CSS. Currently, I think it is only used by GLPI's theme manager to determine which themes are dark.
A good alternative is to use a selector with the data-glpi-theme-dark="1" that GLPI places on the `html` element when you switch to a dark theme.